lib/book_settings.py

```diff
@@ -32,32 +32,54 @@ class BookSettings(object):
 
     COMMON_WORDS = 'the of de'.split()
 
-    def __init__(self, db, book_id, connection):
+    AMAZON_ASIN_PAT = re.compile(r'data\-asin=\"([a-zA-z0-9]+)\"')
+
+    def __init__(self, db, book_id, gConnection, aConnection):
         self._db = db
         self._book_id = book_id
-        self._connection = connection
+        self._gConnection = gConnection
+        self._aConnection = aConnection
 
         book_path = self._db.field_for('path', book_id).replace('/', os.sep)
 
         self._prefs = JSONConfig(os.path.join(book_path, 'book_settings'), base_path=self.LIBRARY)
```

Comment:
I changed it because there's a getter and I figured I'd just use it because I was using it some places but not others. thought it looked nicer this way